website/api/zkvm/quickstart.md

```diff
@@ -5,96 +5,74 @@ slug: ./quickstart
-
-Proceeding with the example above, generating proofs locally would be achieved by running:
+Once you've reached a point where you're ready to generate real proofs, you can do so by setting the `RISC0_DEV_MODE=false`. We recommend that you specify the feature flag `disable-dev-mode`, which will ensure that dev-mode is not accidentally enabled. Generating proofs locally would be achieved by running the following:
 
 ```bash
 RISC0_DEV_MODE=false cargo run --release --features disable-dev-mode
 ```
 
-Note that `RISC0_DEV_MODE=false` is the default behavior, so you can also simply run:
-
-```bash
-cargo run --release --features disable-dev-mode
-```
-
-We would always recommend using `disable-dev-mode` in production, as it ensures that dev-mode is not accidentally enabled.
-
-Also, note that since now proofs are being generated, the execution time of your project will be significantly longer than when running in dev-mode. You might want to consider using [Bonsai] to generate proofs remotely, as it will likely be faster than running proofs locally.
-
-## 6. Local & Remote Proving
-
-You can build and run your zkVM applications using your own hardware, or you can upload your [guest program] to [Bonsai] and make requests for proof generation as needed.
-
-### Local Proving
-
-To run the zkVM on your own machine, we recommend at least 16GB of RAM.
-The readme files on the [zkVM demo applications] show `cargo` commands for local proof generation.
-
-### Remote Proving
-
-To run the zkVM remotely using [Bonsai], [request access] and set the environment variables `BONSAI_API_KEY=<YOUR_API_KEY>` and `BONSAI_API_URL=<BONSAI_URL>`.
-Additional information is available in the [starter template](https://github.com/risc0/risc0/tree/main/templates/rust-starter#running-proofs-remotely-on-bonsai)
+Note that since proofs are now being generated, the execution time of your project will be significantly longer than when running in dev mode. Consider using [Bonsai] to generate proofs remotely, as it will likely be significantly faster than running proofs locally. You can [request access] to Bonsai to set additional flags.
 
 ### Executor Statistics
 
 To gain insights into your application's performance, you can obtain executor statistics by setting the `RUST_LOG` environment variable to `"executor=info"`.
 
-Setting this filter will print statistics about the execution before proof generation, so you can get a sense of how computationally expensive your application is. Since the statistics concern only the executor phase, it is recommended to run your application in dev-mode to avoid the overhead of proof generation:
+Setting this filter will print statistics about the execution before proof generation, so you can understand how computationally expensive your application is. Since the statistics concern only the executor phase, it is recommended to run your application in dev mode to avoid the overhead of proof generation:
```

Comment:
```suggestion
Setting this filter will print statistics about the execution before proof generation, so you can understand how computationally expensive your application is. Since the statistics concern only the executor phase, it is recommended to run your application in dev-mode to avoid the overhead of proof generation:
```